### PR TITLE
 os/board/rtl8730e: SPI multi-slave implementation

### PR DESCRIPTION
**Pull request by kaikaikaime**
Do note that the SPI0 cs1 pin is currently set as PB31, SPI1 set as PB29*

* Just randomly selected pin

### CODE REVIEW

**Review by abhishek-samsung — APPROVED** [review]

File: os/arch/arm/src/amebasmart/amebasmart_spi.c [hunk 1]
```diff
@@ -211,10 +216,13 @@ static struct amebasmart_spidev_s g_spi0dev = {
...
-	.spi_mosi = PB_29,
-	.spi_miso = PB_30,
-	.spi_sclk = PB_31,
-	.spi_cs = PC_0,
+	.spi_mosi = PB_4,
+	.spi_miso = PB_3,
+	.spi_sclk = PB_6,
+	.spi_cs0 = PB_5,
+#if defined(CONFIG_SPI_CS) && defined(CONFIG_AMEBASMART_SPI0_CS)
+	.spi_cs1 = PB_31,
```
Comment (sunghan-chang) [inline os/arch/arm/src/amebasmart/amebasmart_spi.c:224]: This means that we can add just 1 more slave?

Comment (kaikaikaime) [reply os/arch/arm/src/amebasmart/amebasmart_spi.c:224]: Yes, current PR is an addition of 1 slave.

Comment (sunghan-chang) [reply os/arch/arm/src/amebasmart/amebasmart_spi.c:224]: Is there any reason to support only 1 additional slave? I mean even I merge this as it is now, can we add more slave later? Is there any hw or sw limitation on this?

Comment (kaikaikaime) [reply os/arch/arm/src/amebasmart/amebasmart_spi.c:224]: Theoretically there is no HW limit, with more devices, capacitance and length in wires may start to affect signal integrity. My recommendation is a maximum of 3 slaves with testing for stability. As you can observed, to add new slaves, additional pins are necessary, a point to consider.
You most certainly add move slaves in SW similar to the modification done in the PR.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**